assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -281,5 +304,10 @@ describe( 'modules/analytics properties', () => {
 				expect( foundProperty ).toEqual( undefined );
 			} );
 		} );
+		describe( 'getPrimaryPropertyType', () => {
```

Comment:
We also need to add a test (one is enough) that checks that this selector returns the correct type from the state object after it has been changed.

Reply:
@eugene-manuilov  Sounds great. I wasn't sure what to put here because `setPrimaryPropertyType` was testing everything above (the test felt redundant), but (as per your suggestion) I'll change the above test to check the state directly

And then this test can check the selector reads the state after it has changed :+1: